boards/DIM/Src/main.c

```diff
@@ -516,6 +517,8 @@ void RunTask100Hz(void const *argument)
     /* Infinite loop */
     for (;;)
     {
+        App_SharedStateMachine_Tick(state_machine);
+        Io_SevenSegDisplays_WriteCommands();
```

Comment:
Could possibly define this as a callback function that  gets called at the end of `App_SevenSegDisplays_SetHexDigits`. Haven't really decided if it's better that way though.

Reply:
Yeah I think registering it at a callback makes more sense.

Reply:
Registered a callback function that gets invoked after we set a value to the 7-seg displays (e.g. `SetHexDigits` or `SetUnsignedBase10Value`)